cli/tests/unit/test_marathon_pod.py

```diff
@@ -57,7 +57,7 @@ def resource_reader(path):
 
 def _assert_pod_remove_invoked_successfully(pod_id, force):
     def resource_reader(path):
-        pass
+        assert False, "should not be called"
```

Comment:
If I do that, I get this error from the syntax tests:

```
tests/unit/test_marathon_pod.py:60:9: F999 assertion is always true, perhaps remove parentheses?
tests/unit/test_marathon_pod.py:72:9: F999 assertion is always true, perhaps remove parentheses?
flake8.main.application   MainProcess   1770 INFO     Found a total of 12 violations and reported 2
ERROR: InvocationError: '/home/charles/repos/dcos/dcos-cli/cli/.tox/py34-syntax/bin/flake8 --verbose dcoscli tests setup.py'
```

I think `assert` is actually an element of Python syntax, so putting parentheses around its arguments just means we're `assert`ing on a tuple, instead of passing two arguments to `assert`.

Yep: https://docs.python.org/3.4/reference/simple_stmts.html#grammar-token-assert_stmt